src/XRP/xrp-transaction-type.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Types of transactions on the XRP Ledger.
+ *
+ * This is a partial list. Please file an issue if you have a use case that requires additional types.
```

Comment:
Do we have a publicly available list of types that are on the roadmap?

Reply:
Our plan is basically to add things as there is external demand, due to the limited engineering resources we have versus other priorities.